src/styles.scss

```diff
@@ -203,3 +203,54 @@ body {
   flex-direction: column;
   overflow-x: hidden !important;
 }
+
+.table {
+  width: 100%;
+  table-layout: fixed;
+}
+
+.table td {
+  table-layout: fixed;
+  border: 1px solid #ddd;
+  padding: 8px;
+  text-align: center;
+  vertical-align: middle;
+  align-items: center;
+  justify-content: center;
+}
+
+.table th {
+  border: 1px solid #ddd;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  text-align: center;
+  background-color: #ddd;
+  vertical-align: middle;
+}
+
+.table th:nth-child(1),
+.table td:nth-child(1) {
+  width: 10%;
+}
+.table th:nth-child(2),
+.table td:nth-child(2) {
+  width: 20%;
+}
+.table th:nth-child(3),
+.table td:nth-child(3) {
+  width: 20%;
+}
+.table th:nth-child(4),
+.table td:nth-child(4) {
+  width: 10%;
+  text-align: center;
+}
+.table th:nth-child(5),
+.table td:nth-child(5) {
+  width: 30%;
+  text-align: center;
+}
+.table th:nth-child(6),
+.table td:nth-child(6) {
+  width: 10%;
+}
```

Comment:
We should use tailwind classes for such simple styles. All the above classes should be unnecessary

e.g. `<table class="w-full table-fixed">`
e.g. `<th class="w-[30%] text-center">`